frontend/src/components/LoxotronCasinoLogoComponent/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface LogoProps {
+    width: number,
+    height: number
+}
+
+export const LogoComponent: React.FC<LogoProps> = ({ width = 300, height = 300 }) => {
+    return (
+        <svg width={width} height={height} viewBox="0 0 500 500" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think it's better to store it as a separate svg file in [assets/svg](https://github.com/dkushche/Loxotron/tree/master/frontend/src/assets/svg) as we did with `LoxotronCasinoText`